src/bereikbaarheid/resources/utils.py

```diff
@@ -107,15 +107,22 @@ def refresh_materialized(db_table: str):
 # -------------------------------------------
 
 
-def convert_to_date(date: str = None, format: str = "%d/%m/%y %H:%M"):
+def convert_to_date(date: str = None, format: str = "%d/%m/%y %H:%M") -> datetime:
     """Convert string format to datetime"""
 
+
     try:
         _date = datetime.datetime.strptime(date, format)
+        return _date
     except ValueError:
-        print(f"verkeerd datumformat voor {date}, gewenst format is {format}")
+        try:
+            format = "%Y-%m-%d %H:%M:%S.%f"
+            _date = datetime.datetime.strptime(date, format)
+            return _date
+        except ValueError:
+            print(f"verkeerd datumformat voor {date}, gewenst format is {format}")
 
-    return _date
+    return ValueError
```

Comment:
Waarom return je hier een ValueError? Ik denk dat wat je wil, een `raise ValueError` in de except wil.